src/test/java/jalau/at18/katas/game2048/team1/MainTest.java

```diff
@@ -0,0 +1,141 @@
+package jalau.at18.katas.game2048.team1;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class MainTest {
+    private final int[][] exampleMatrix = {{0, 0, 0, 0}, {0, 0, 0, 0}, {0, 0, 2, 0}, {0, 0, 0, 0}};
+    private final int[][] matrixGameOver = {{1, 2, 3, 4}, {5, 6, 7, 8}, {9, 1, 2, 3}, {4, 5, 9, 7}};
+    private final int[][] matrixGameNoOver = {{1, 2, 3, 4}, {5, 6, 7, 0}, {9, 1, 2, 4}, {4, 5, 9, 20}};
```

Comment:
This is ok, however, a small detail about these matrices is that they contain values that are not valid for game, like 9, 20, 5, 7 and 6. Those will never appear in the game accoring to game dynamics.

The point is that tests should reflect the actual behavior we expect, and probably if we would have a method that validates that all values are correct using these matrices, it would make sense to have them, but in this case, it would be better to just use values that we expect to see in the matrices

0, 2, 4, 8, 16, 32, 64, 128, 254, 512, 1024 and 2048

Reply:
We already correct these matrix